libs/ui-react/src/lib/Components/Link/Link.tsx

```diff
@@ -0,0 +1,146 @@
+ *   Useful for creating router links or other semantic elements with link appearance.
+ * @param {string} [className] - Additional custom CSS classes to apply. Do not use this prop to modify the component's core appearance - use the `appearance` prop instead.
+ * @param {React.ReactNode} [children] - The link's content, typically text.
+ * @param {React.AnchorHTMLAttributes<HTMLAnchorElement>} [...] - All standard anchor props (e.g., `href`, `onClick` etc.).
+ *
+ * @warning The `className` prop should only be used for layout adjustments like margins or positioning.
+ * Do not use it to modify the link's core appearance (colors, padding, etc). Use the `appearance` prop instead.
+ *
+ * @example
+ * import { Link } from '@ldls/ui-react';
+ *
+ * // Basic underlined link
+ * <Link appearance="underlined" size="md" href="/page">
+ *   Go to Page
+ * </Link>
+ *
+ * // Accent button-like link with icon
+ * import { ArrowRight } from '@ldls/ui-react/symbols';
+ *
+ * <Link appearance="accent" size="sm" href="https://example.com" isExternal icon={ArrowRight}>
+ *   External Site
+ * </Link>
+ *
+ * // Link as a router link (asChild pattern)
+ * import { Link as RouterLink } from 'react-router-dom';
+ *
+ * <Link asChild appearance="underlined" size="md">
+ *   <RouterLink to="/dashboard">Dashboard</RouterLink>
+ * </Link>
+ *
+ * // Note: When using asChild, the child element is responsible for its own content.
+ * // Icons and other Link props like 'icon' are ignored when asChild is true - handle these in the child if needed.
+ */
+export const Link = React.forwardRef<HTMLAnchorElement, LinkProps>(
+  (
+    {
+      className,
+      children,
+      appearance,
+      size = 'md',
+      icon,
+      isExternal = false,
+      asChild = false,
+      ...props
+    },
+    ref,
+  ) => {
+    const iconSizeMap: { [key: string]: IconSize } = {
+      sm: 16,
+      md: 20,
+    };
+
+    const calculatedIconSize = size ? iconSizeMap[size] : 20;
+    const IconComponent = icon;
+
+    const Comp = asChild ? Slot : 'a';
+
+    return (
+      <Comp
+        ref={ref}
+        className={cn(
+          className,
+          linkVariants({
+            appearance,
+            size,
+          }),
+        )}
+        target={isExternal && !asChild ? '_blank' : undefined}
+        rel={isExternal && !asChild ? 'noopener noreferrer' : undefined}
+        {...props}
+      >
+        {asChild ? (
+          children
+        ) : (
+          <>
+            {IconComponent && (
+              <IconComponent
+                size={calculatedIconSize}
+                className="flex-shrink-0"
+              />
+            )}
+            <span className="min-w-0 truncate">{children}</span>
+            {isExternal && (
+              <ExternalLink
+                size={calculatedIconSize}
+                className="flex-shrink-0"
+              />
```

Comment:
```suggestion
              <>
				<ExternalLink
				size={calculatedIconSize}
				className="flex-shrink-0"
				aria-hidden="true"
				/>
				{/* Hidden text for screen readers */}
				<span className="sr-only">(opens in a new tab)</span>
              </>
```

would be nice to add  opens in a new tab to screen readers as the icon is not informative